server/src/main/java/org/opensearch/cluster/metadata/AliasMetadata.java

```diff
@@ -259,7 +259,7 @@ public static Diff<AliasMetadata> readDiffFrom(StreamInput in) throws IOExceptio
 
     @Override
     public String toString() {
-        return Strings.toString(XContentType.JSON, this, true, true);
+        return org.opensearch.common.Strings.toString(XContentType.JSON, this, true, true);
```

Comment:
```suggestion
        return Strings.toString(XContentType.JSON, this, true, true);
```

Reply:
This one can't be moved yet w/o `opensearch-core` library taking a dependency on the `opensearch-x-content` library.  I'm keeping those decoupled. I have a follow up that further decouples this, though.